AppInsightsCommon/Util.ts

```diff
@@ -674,4 +682,33 @@ export class AjaxHelper {
             name: name
         };
     }
-}
\ No newline at end of file
+}
+
+/**
+ * A utility class that helps getting time related parameters
+ */
+export class dateTime {
+    /**
+     * Get the number of milliseconds since 1970/01/01 in local timezone
+     */
+    public static Now = (window.performance && window.performance.now && window.performance.timing) ?
+        function () {
+            return window.performance.now() + window.performance.timing.navigationStart;
```

Comment:
From https://developer.mozilla.org/en-US/docs/Web/API/Performance/now:

> The values returned by Performance.now() always increase at a constant rate, independent of the system clock (which might be adjusted manually or skewed by software like NTP). Otherwise, performance.timing.navigationStart + performance.now() will be approximately equal to Date.now().

Reply:
From [https://developer.mozilla.org/en-US/docs/Web/API/Performance/now](https://developer.mozilla.org/en-US/docs/Web/API/Performance/now):

> The values returned by Performance.now() always increase at a constant rate, independent of the system clock (which might be adjusted manually or skewed by software like NTP). Otherwise, performance.timing.navigationStart + performance.now() will be approximately equal to Date.now().

---
In reply to: [213844325](https://github.com/Microsoft/ApplicationInsights-JS/pull/660#discussion_r213844325) [](ancestors = 213844325)